src/libraries/System.DirectoryServices.Protocols/ref/System.DirectoryServices.Protocols.cs

```diff
@@ -382,6 +382,8 @@ public partial class LdapSessionOptions
         internal LdapSessionOptions() { }
         public bool AutoReconnect { get { throw null; } set { } }
         public string DomainName { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("windows")]
+        public string TrustedCertificatesDirectory { get { throw null; } set { } }
```

Comment:
It looks like we expose TrustedCertificatesDirectory in the `netstandard2.0` assembly.  Then on .NETFramework we expect that to be replaced with the inbox assembly from netfx which doesn't have this API.

We may want to ifdef this `#if NET` to only expose the API on .NET versions where it is implemented.

API didn't catch this because we don't compare netstandard2.0 to netframework if a package doesn't have an assembly for it.  cc @ViktorHofer maybe we should add a comparison tuple when we see the empty folder?